odoo/models.py

```diff
@@ -227,6 +227,7 @@ class BaseModel(object):
 
     _name = None                # the model name
     _description = None         # the model's informal name
+    _is_business = False        # should be True for important business models
```

Comment:
main_business maybe. Main does not mean anything; ir.model is a main model. business is the key word.